test/graph/graph-mary-manipulation-test.js

```diff
@@ -8,27 +8,34 @@ var maryStream = testStreams.fullList.filter(function(stream) {
 
 describe('Graph manipulation for `mary` stream', function() {
     var graph;
-    beforeEach(function(done) {
+    function createGraph(done) {
```

Comment:
Why was the `beforeEach` removed from here and copied into all the tests below? Even the nested `describe`s should work properly with the `beforeEach` here.